### DISCUSSION TITLE
where is https://github.com/PABCSoft?

### DISCUSSION THREAD

**Discussion by retsyo** [pascalabcnet/pascalabcnet#3233] [Q&A]
some projects on https://forum.mmcs.sfedu.ru/t/proekty-na-pascalabc-net/2272/111 point to https://github.com/PABCSoft which is 404 now

so what is the new URL?

Thanks

**Comment by SunSerega** [comment]
If they were moved - you'd get automatically redirected by GitHub. So they are probably completely deleted now.

Is there anything in particular you wanted an example project for, or were you just looking around?
And why ask questions here instead of on the forum page in question...
